### PR TITLE
GALE Arabic and Mandarin recipes

### PR DESCRIPTION
**Pull request by desh2608**
This PR adds capability to prepare manifests for GALE Arabic and Mandarin datasets, both of which are broadcast conversation data. Since the datasets do not come with official splits, we use the Kaldi splits for both.

### CODE REVIEW

File: lhotse/recipes/gale_mandarin.py [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+        )
+        df['text'] = df['text'].apply(lambda x: x.strip() if not pd.isnull(x) else x)
+        for idx, row in df.iterrows():
+            supervision_id = f"{row['reco_id']}-{row['speaker']}-{idx}"
+            duration = round(row['end'] - row['start'], ndigits=8)
+            if supervision_id in supervision_ids or duration <= 0:
+                continue
+            supervision_ids.add(supervision_id)
+            supervisions.append(
+                SupervisionSegment(
```
Comment (pzelasko) [inline lhotse/recipes/gale_mandarin.py:214]: I think you could add “language” too

File: lhotse/recipes/gale_arabic.py [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+        )
+        df['text'] = df['text'].apply(lambda x: x.strip() if not pd.isnull(x) else x)
+        for idx, row in df.iterrows():
+            supervision_id = f"{row['reco_id']}-{row['speaker']}-{idx}"
+            duration = round(row['end'] - row['start'], ndigits=8)
+            if supervision_id in supervision_ids or duration <= 0:
+                continue
+            supervision_ids.add(supervision_id)
+            supervisions.append(
+                SupervisionSegment(
```
Comment (pzelasko) [inline lhotse/recipes/gale_arabic.py:224]: I think you could add “language” too

**Review by pzelasko — COMMENTED** [review]
Great job! I left one comment. Let me know when it’s good to merge.

**Comment by desh2608** [comment]
@pzelasko I have added the `language` tag in supervisions. It can be merged now.

**Comment by pzelasko** [comment]
Thanks!

**Pull request merged**